utils/print-deployments-logs.js

```diff
@@ -105,16 +85,17 @@ module.exports = async ({ github, context, fs, customDomain }) => {
     await Promise.all(delPromises);
   };
 
-  const mergeExistingPRComments = async () => {
-    let commentBody = alignRight(`${GMTConverter(defaultBody)}\n`);
+  async function mergeExistingPullRequestComments() {
+    let commentBody = alignRight(`${(defaultBody)}\n`);
     botCommentsArray.forEach(({ body }) => {
-      commentBody = commentBody + alignRight(`${GMTConverter(body)}\n`);
+      commentBody = commentBody + alignRight(`${(body)}\n`);
     });
-    await createNewPRComment(commentBody);
-    await deleteExistingPRComments();
-  };
+    commentBody = sortComments(commentBody);
```

Comment:
I added sorting for the commits because this was removed when we updated the schema to just commit hashes.